src/util/config/build.js

```diff
@@ -25,6 +25,21 @@ export const schema = {
 			arg: 'asset-port',
 			env: process.env.NODE_ENV !== 'test' && 'ASSET_SERVER_PORT', // don't read env in tests
 		},
+		protocol: {
```

Comment:
Oh, maybe you mean in `index.js`? I think it might make sense to add some validation in there, but I still think it needs to get added to the schema.